Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM semtech/mu-javascript-template:1.4.0
-LABEL maintainer="madnificent@gmail.com"
+FROM semtech/mu-javascript-template:1.5.0-beta.1
```

Comment:
Is there a reason why `beta.1` and not `beta.2` ?

Reply:
there is a beta -2now? will update :-)